sky/shell/BUILD.gn

```diff
@@ -96,6 +96,10 @@ if (is_android) {
              ":jni_headers",
              ":common",
            ]
+    ldflags = [
```

Comment:
I'm not sure if this is the right place to put these flags, but seems reasonable.